example/src/main/java/com/uploadcare/android/example/viewmodels/UploadViewModel.kt

```diff
@@ -142,10 +141,12 @@ class UploadViewModel(application: Application) : AndroidViewModel(application)
             if (result.uploadcareFile != null) {
                 showProgressOrResult(false, result.uploadcareFile.toString())
             } else if (result.backgroundUploadUUID != null) {
-                showProgressOrResult(false,
-                        "Uploading in background: ${result.backgroundUploadUUID.toString()}")
+                showProgressOrResult(
+                    false,
+                    "Uploading in background: ${result.backgroundUploadUUID.toString()}"
+                )
 
-                if(result.backgroundUploadUUID != backgroundUploadUUID.value){
+                if (result.backgroundUploadUUID != backgroundUploadUUID.value) {
                     backgroundUploadUUID.value = result.backgroundUploadUUID
```

Comment:
**backgroundUploadUUID** is nullable in **UploadcareWidgetResult**, but **backgroundUploadUUID.value** expects non-nullable value

Reply:
We can fix it in another task, but you have to create the task and place todo and task reference here